src/FooBarFrontend/common/core/aurelia-enhancer.ts

```diff
@@ -124,7 +126,12 @@ export function enhance(au: Aurelia, viewModel: any, element: HTMLElement, parti
         $(document).trigger('vk-aurelia-enhanced', <AureliaEnhancedEventArgs>{ isPartial: partial, element: element });
     }
 
-    let startPromise = <Promise<void>>au.enhance({ host: element, component: viewModel }).start();
+    let startPromise = <Promise<void>>aurelia.enhance({
+        host: element,
+        component: viewModel,
+        //@ts-ignore
+        container: aurelia.container
```

Comment:
This line should be omitted, or changed to: `aurelia.container.createChild()`

Reply:
The container is the own container of the custom element created by the enhancement. Can you have a look at the doc here https://docs.aurelia.io/getting-to-know-aurelia/app-configuration-and-startup#enhance and create an issue in the Aurelia repo if it's unclear?